### ISSUE TITLE
Clarification of ENVO ontologies

### ISSUE THREAD

**Issue by ljohnson09** [microbiomedata/issues#683]
**Deliverable this task is associated with**

_See Deliverables tab here: _

- [Add the Deliverable #]
4.1 Perform user research to refine the Data Portal user experience

**RACI**

_Tag people in their roles_
- Responsible: @turbomam @mslarae13 @cmungall @aclum 
- Accountable: @aclum 
- Consulted:
- Informed:

**Describe the the task**
- [ ] Users were confused by hierarchy, did not know what all of the ontologies meant. Note from Alicia: These fields are GSC fields populated w/ ENVO terms but we also use Plant and Uberon ontologies; Maybe link out to an FAQ about ontologies. Can we use the EBI website instead of Ontobee?  

**Criteria for completion**
- [What does success look like for this task to be considered complete?]

**Estimate people time**
- [Hours or days of people time. 1 person, 4 hours]

**Completion Date (Goal)**
- [GOAL!]

**Target Sprint Start & End Dates**
- Start:
- End:

**Tag Blocker/Contingent upon issues**
- [Tagg issues]

**Comment by aclum** [comment]
@turbomam  we talked about this several months during the schema freeze but wanted to revisit.  Do we want a grouping slot like we have for [gold_path_field](https://microbiomedata.github.io/nmdc-schema/gold_path_field/) with a tooltip annotation or should I have this directly added in the nmdc_schema code. I believe the hold up when we discussed this before is that env_broad_scale, env_local_scale, and env_medium are mixs slots.

**Comment by turbomam** [comment]
yes, but I have injected stuff into src/schema/mixs.yaml before, so I can do that again here

**Comment by aclum** [comment]
@turbomam Do you have time for that this sprint?

**Comment by mslarae13** [comment]
I am not sure I understand the ask here .
Is this for the submission portal? If so I'd like more context about the ask and change before charging ahead.
Also, I don't think @turbomam should pick this up this sprint. We need to finish the env triad squad.

**Comment by aclum** [comment]
This is for the data portal to get a tooltip (from nmdc-schema) for the environmental context fields like we added for the GOLD ecosystem classifications. It should be straightforward but is more complicated than what we did for GOLD b/c these are MIxS slots so not sure how to do this myself.
